spharpy/spherical.py

```diff
@@ -271,9 +279,21 @@ def spherical_harmonic_basis(n_max, coords):
     ----------
     n_max : integer
         Spherical harmonic order
-    coordinates : :doc:`pf.Coordinates <pyfar:classes/pyfar.coordinates>`
-        Coordinate object with sampling points for which the basis matrix is
+    coordinates : :doc:`pf.Coordinates <pyfar:classes/pyfar.coordinates>` or
+        `sp.SamplingSphere <spharpy:classes/spharpy.samplings.coordinates>`
+        objects with sampling points for which the basis matrix is
         calculated
+    normalization : str, optional
+        Normalization convention, either 'n3d', 'maxN' or 'sn3d'.
+        The default is 'n3d'.
+        (maxN is only supported up to 3rd order)
```

Comment:
Since this is only valid up to third order we should raise an error on invalid inputs.

Reply:
done